### ISSUE TITLE
Dialogs Text and Answer fields: text editor flickering

### ISSUE THREAD

**Issue by rezeau** [h5p/h5p-dialogcards#59]
When entering text in the Text and Answer fields, the editor provides a paragraph formatting drop-down list. This list actually contains a single item, "Normal", which is not terribly useful. More options could be provided.
Anyway the current bug is that when clicking on that "Normal" drop-down list in the editor menu, all we get is flickering, as show in the attached gif file.
This is a strange bug, as in the very similar Flashcards H5P activity, in the Tip Text editor we get the same buttons but the (useless) "Normal" drop-down list is OK (i.e. it does not flicker).
Note.- On the attached video capture the flickering is much slower than in reality.
![2019-07-15_17-03-55](https://user-images.githubusercontent.com/503583/61227115-f6557380-a723-11e9-82b5-d6c0eeed09ad.gif)

**Comment by otacke** [comment]
Hi @rezeau !

I think this is a [known issue](https://h5ptechnology.atlassian.net/browse/HFP-2282).

Best,
Oliver

**Comment by rezeau** [comment]
Well, actually I can reproduce this bug **everywhere**:
on the H5P site
on Drupal
on WordPress
on Moodle

**Comment by otacke** [comment]
@rezeau 
You're correct. Thanks for amending the ticket. My message was: The problem is known, not that it has been perfectly investigated.

**Comment by maxmoeschinger** [comment]
Hi, I am still having this issue. When do you plan on fixing it?

**Comment by gemguardian** [comment]
We have replicated this issue in Lumi as well.
